Assets/NuGet/Editor/NugetHelper.cs

```diff
@@ -1154,10 +1155,13 @@ public static bool Install(NugetPackage package, bool refreshAssets = true)
                         if (refreshAssets)
                             EditorUtility.DisplayProgressBar(string.Format("Installing {0} {1}", package.Id, package.Version), "Downloading Package", 0.3f);
 
-                        Stream objStream = RequestUrl(package.DownloadUrl, package.PackageSource.ExpandedPassword, timeOut: null);
-                        using (Stream file = File.Create(cachedPackagePath))
+                        Stream objStream = RequestUrl(package.DownloadUrl, package.PackageSource.UserName, package.PackageSource.ExpandedPassword, timeOut: null);
+                        if (objStream != null)
```

Comment:
>  if (objStream != null) [](start = 22, length = 24)

Is this expected, or an error? Maybe print like a warning?

Reply:
Oops, a remnant from a different attempted implementation of RequestUrl.